docs/content/config/advanced/optional-config.md

```diff
@@ -11,7 +11,6 @@ This is a list of all configuration files and directories which are optional or
 - **sieve-filter:** directory for sieve filter scripts. (Docs: [Sieve][docs-sieve])
 - **sieve-pipe:** directory for sieve pipe scripts. (Docs: [Sieve][docs-sieve])
 - **opendkim:** DKIM directory. Auto-configurable via [`setup.sh config dkim`][docs-setupsh]. (Docs: [DKIM][docs-dkim])
-- **ssl:** SSL Certificate directory. Auto-configurable via [`setup.sh config ssl`][docs-setupsh]. (Docs: [SSL][docs-ssl])
```

Comment:
Dropping `Auto-configurable via [`setup.sh config ssl`][docs-setupsh].` okay. Is this directory no longer relevant to document?

Is there no way this directory is generated/used now?

The referenced doc page does have some content referencing it [and the command that is being removed](https://docker-mailserver.github.io/docker-mailserver/edge/config/security/ssl/#self-signed-certificates). While the command may not be relevant, the section still is. Note that I have documented the generation of our test certs [here](https://github.com/docker-mailserver/docker-mailserver/tree/master/test/test-files/ssl/example.test) which could be adapted to this section of the docs.

You've also missed removing the `docs-ssl` link at the bottom of the file as this is the only line referencing it, should this line remain removed entirely.

Reply:
I must have overlooked that. I will fix that.